src/styles/star.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background: url(../images/star.svg) no-repeat center;
+    display: flex;
+
+    &:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  &--1 &__star:nth-child(1),
+  &--2 &__star:nth-child(-n + 2),
+  &--3 &__star:nth-child(-n + 3),
+  &--4 &__star:nth-child(-n + 4),
+  &--5 &__star:nth-child(-n + 5) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
Your solution for displaying the active stars is correct, but it could be optimized. You can use a loop to avoid repetition. Consider using `@for` loop from Sass. It would look something like this: 
scss
@for $i from 1 through 5 {
  &--#{$i} &__star:nth-child(-n + #{$i}) {
    background-image: url(../images/star-active.svg);
  }
}
``` 
This will generate the same CSS as your current code, but it's more maintainable and readable.